Dynatrace/dynatrace-operator
# Remove unnecessary roles from CSI serviceAccount

## Description

[K8S-10892](https://dt-rnd.atlassian.net/browse/K8S-10892)

Remove unnecessary roles from CSI serviceAccount
They were probably copy-pasted from the Operator or some example, and nobody questioned them.

## How can this be tested?

deploy Operator and a DynaKube that uses the CSI-driver, nothing should break. (no actual change in behaviour)
